ixwebsocket/IXWebSocketTransport.cpp

```diff
@@ -68,7 +68,9 @@ namespace ix
     const int WebSocketTransport::kDefaultPingIntervalSecs(-1);
     const int WebSocketTransport::kDefaultPingTimeoutSecs(-1);
     const bool WebSocketTransport::kDefaultEnablePong(true);
```

Comment:
I don't know if this a 100 ms makes sense. In real life it's likely that it will take longer.

Reply:
We could set a greater value